webapp-django/crashstats/signature/views.py

```diff
@@ -424,3 +424,112 @@ def signature_graph_data(request, channel):
     )
 
     return data
+
+
+@waffle_switch('signature-report')
+def signature_summary(request):
+    '''Return a list of specific aggregations. '''
+    params = get_validated_params(request)
+    if isinstance(params, http.HttpResponseBadRequest):
+        # There was an error in the form, let's return it.
+        return params
```

Comment:
I know I'm guilty of this pattern too, but a more pythonic thing would be to change `get_validated_params` to raise a `ValidationError(problem)` or something and change these lines to:
```python
try:
    params = get_validated_params(request)
except ValidationError as exc:
    return http.HttpResponseBadRequest(exc)
```